### PR TITLE
Jswope00/assimilate legacy docs

### PR DESCRIPTION
**Pull request by jswope00**
@sarina I am heading towards this approach for assimilating the legacy edx docs into the Open edX docs. I think the approach fits the current structure of Open edX docs, it has a nice "tracking" system in that docs that are removed from migration_wip are done, and it is easy enough that others can be trained on it. 

The process is as follows: 

1. Categorize legacy pages into pages according to their diataxis purpose. (e.g. "Exploring Your Dashboard and Profile" becomes "What is the Course Dashboard?" (concept), "Add Profile Information" (how-to). Split larger pages as necessary. 

Pages that are obviously EdX.org specific and irrelevant to Open edX (e.g. The edX Partner Portal) can be deleted and excluded from the migration. Pages that are ambiguous should be migrated and we'll make a determination on them later. 

2. Give every document a diataxis tag. You can also use the tag "navigational" if the document only serves a navigational purpose. 
3. Give every document a role tag(s). I haven't done that in these docs yet, but I think it's a good idea. 

4. Add ".. seealso::" links, based on other documents that belong with the one you are editing. This is sometimes related docs from a larger doc that you broke up, and sometimes it is documents from other places that weren't previously attached. Therefore, this requires some critical thinking, knowledge of Open edX, and knowledge of the docs. I suggest in a future phase we revisit these "see also" links with a more systematic approach, once all docs are in the Open edX docs format. 

5. Fix any errors that arise in build. Common ones:
- Update image links so they don't break.
- Update :ref: :include:, and :doc: links so they don't break. 

# Questions/Issues
1. I am wary (not worried, just aware of it) we are making the docs less readable and harder to maintain. 
Less Readable = Breaking up cohesive docs into pieces and then linking to each other (instead of in one doc). Maybe that is necessary for now, and as we improve we'll strip away unsupported and redundant docs, and get tighter with our "see also" linking. 
Harder to Maintain = I find myself creating more absolute references via :doc:. I'll try to use more :ref: in future commits. 

2. What is the function of a "reusable_content" folder? How does it help organization? Isn't everything potentially re-usable? What if we think something won't be re-used, but then it is, or vice versa?

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jswope00!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@feanil`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Review by sarina — COMMENTED** [review]
This looks good to me. I didn't know if you're looking for me to review with an eye to edx.org references. I also (forgive me) don't know the advantages of using `:ref:` over `:doc:` - I find I use `:doc:` mostly.

**Comment by jswope00** [comment]
> This looks good to me. I didn't know if you're looking for me to review with an eye to edx.org references. I also (forgive me) don't know the advantages of using `:ref:` over `:doc:` - I find I use `:doc:` mostly.

I advise a move to :ref: links throughout. :doc: links use path references, so they break if you move a file. :ref: links do not break if you move the file around. And I'm not aware of any other advantage of :doc: so I'm changing to :ref: whenever I find it.

File: source/community/index.rst
Comment (jswope00) [reply source/community/index.rst:9]: @sarina It came from Legacy docs but it is very much a community item more than an educator item. It talks about subscribing to different types of announcements not just educator ones.

File: source/educators/concepts/index.rst
Comment (jswope00) [reply source/educators/concepts/index.rst:10]: It isn't empty. I points to an index file that then points to three ID docs.

File: source/educators/concepts/open_edx_platform/what_is_course_dashboard.rst
Comment (jswope00) [reply source/educators/concepts/open_edx_platform/what_is_course_dashboard.rst:32]: Haphazardly, as it is noticed. Looking thoroughly is part of the migration cleanup checklist.

**Comment by jswope00** [comment]
@sarina here is a large chunk of the docs that I'd like to get merged into main so that other members of the Documentation WG can start working with them. There are still 5 more sections to move (8-12) and those will come later. It makes the educators TOC pages (esp. the How-To page) a bit disorganized but the UI/UX part is coming  up very soon and nothing is *wrong*, just disorganized.

**Comment by sarina** [comment]
Thanks for addressing my previous comments. I'll take a look at this by end of day tomorrow. Let me know if this is ready to merge, if I don't find problems with it.

**Comment by jswope00** [comment]
If you don't find any problems, then this is ready to merge.

File: source/_tags/tagsindex.rst
Comment (sarina) [inline source/_tags/tagsindex.rst:1]: Question: Why are we collecting orphan pages? My assumption is because this is a strict migration, and we can clean up things tagged "orphan" at a later point

File: source/educators/concepts/accessibility/index.rst
Comment (sarina) [inline source/educators/concepts/accessibility/index.rst:4]: Nit: I would call this something like "Accessibility in Course Creation" to distinguish it from the pages on developing an accessible product. If you agree I can change later on

File: source/educators/concepts/accessibility/supporting_learners_diverse_needs.rst
Comment (sarina) [inline source/educators/concepts/accessibility/supporting_learners_diverse_needs.rst:91]: These need a blank line in between each to render properly. (or put in a bulleted list)

image

File: source/educators/concepts/open_edx_platform/index.rst
Comment (sarina) [inline source/educators/concepts/open_edx_platform/index.rst:1]: I think this title is wrong

**Review by sarina — APPROVED** [review]
I did some spot checking and found a couple problems, but nothing that will stop me from merging this right now so we can move along. I reviewed all the index files. It looks good to me!

**Pull request merged**
